### ISSUE TITLE
build command does not allow key specification

### ISSUE THREAD

**Issue by meisinger** [phonegap-build/pgb-cli#1]
Through the website I am able to select a key and when selected it performs a build. This is not possible through the cli.

Through trial and error, it appears the last key used on the website is what is used for all build requests from the cli. This is based on the fact that if the last used key from the website is locked then the build fails (even if other keys are unlocked). It is only when you go to the website and select a different key will that key be used in any subsequent build requests from the cli.

Unless there is a hidden parameter or argument that I am missing, it appears that the "build" request is missing the ability to select a key or there is a missing "build-with-key" command.

_Assuming that the _ `app[selected_(platform)_key_id]` _ parameter is possible_

**Comment by meisinger** [comment]
It appears the "api" needs to support the ability to perform a "PUT" passing a key (e.g. `app[selected_(platform)_key_id]`) as well as a `platform` in order for this to work.

As it stands right now the "build" command is similar to hitting the "Rebuild" button on the website. I will add an issue over in the "api" project and report back once finished.

**Comment by goya** [comment]
yeah build is just a build request that takes no args.  if you're updating the app (like changing keys etc.) then use:
```
> pgb update 12345 ios-key=12
> pgb update -? # get full list of options available
``` 
be sure to re-install, as the fix is in the pgb-api dependency 1.0.4 - i'll be publishing a new cli that updates the usage of the update command to make this more clear and then i'll close it.

**Comment by goya** [comment]
fixed in v1.0.1
